src/lib/swapper/swappers/ThorchainSwapper/types.ts

```diff
@@ -124,3 +124,12 @@ export type ThorChainId =
   | ThorCosmosSdkSupportedChainId
   | ThorEvmSupportedChainId
   | ThorUtxoSupportedChainId
+
+type ThorNodeStatusResponseSuccess = {
+  // Non-exhaustive, the 'done' status is all we care about here
+  observed_tx: {
+    status: 'done' | 'incomplete'
```

Comment:
This can and will be undefined at some stage
```suggestion
    status?: 'done' | 'incomplete'
```

Reply:
000f8c2d15